components/Listwords/Listwords.tsx

```diff
@@ -1,144 +1,26 @@
-                        </button>
+    <section>
+      {props.error && props.onError()}
+      {props.loading && props.onLoading()}
+
+      {!props.loading && !props.listWords.length && props.onEmptySearch()}
 
-                        <button
-                          disabled={sending}
-                          type="button"
-                          onClick={() =>
-                            handlerUpdateRatingWord({ id: item.id, rating: -1 })
-                          }
-                        >
-                          &#8595;
-                        </button>
-                      </>
-                      : null}
-                  </>
-                )}
-                {item.hideAllText ? (
-                  <>
-                    &bull; &nbsp;
-                    <span>
-                      {showNotes ? item.title : item.text_en}:{" "}
-                      {showNotes ? item.text : item.text_es}
-                      <span
-                        className={styles["Eye"]}
-                        onClick={() => {
-                          handlerList(item, !item.hideAllText);
-                        }}
-                      >
-                        {!showNotes && <span>{(item.rating || 0) + " "}</span>}
-                        <Image
-                          src={eyeOpen}
-                          alt={"alt"}
-                          width={16}
-                          height={16}
-                          layout="intrinsic"
-                        />
-                      </span>
-                    </span>
-                  </>
-                ) : (
-                  <div className={styles["Rendered-Text"]}>
-                    <span>&bull; {showNotes ? item.title : item.text_en}:</span>{" "}
-                    <span
-                      className={styles["Eye"]}
-                      onClick={() => {
-                        handlerList(item, !item.hideAllText);
-                      }}
-                    >
-                      <Image
-                        src={eyeClose}
-                        alt={"alt"}
-                        width={16}
-                        height={16}
-                        layout="intrinsic"
-                      />
-                    </span>
-                    <pre>
-                      {" \n" + (showNotes ? item.text : item.text_es)}
-                      {item.urlImg && (
-                        <div>
-                          <Image
-                            src={item.urlImg}
-                            alt={"alt"}
-                            width={0}
-                            height={0}
-                            layout="responsive"
-                          />
-                        </div>
-                      )}
-                    </pre>
-                  </div>
-                )} */}
-              </li>
-            );
-          })}
-      </ul>
+      {props.listWords.map(props.render)}

```

Comment:
this code should be inside ul?, because the words page is rendering <li> elements in render function and is ending out of <ul> with current design

Reply:
done